tests/basic/sharedv4_svc_test.go

```diff
@@ -45,8 +45,28 @@ var _ = Describe("{Sharedv4SvcPodRestart}", func() {
 			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("nfsserverfailover-%d", i))...)
 		}
 
+		// scale all the apps to have 2 pods since later we will have only 2 nodes where the pods can run
+		for _, ctx := range contexts {
+			Step(fmt.Sprintf("scale app %s to 2", ctx.App.Key), func() {
+				applicationScaleUpMap, err := Inst().S.GetScaleFactorMap(ctx)
+				Expect(err).NotTo(HaveOccurred())
+				for name := range applicationScaleUpMap {
+					applicationScaleUpMap[name] = int32(2)
+				}
+				err = Inst().S.ScaleApplication(ctx, applicationScaleUpMap)
+				Expect(err).NotTo(HaveOccurred())
+			})
```

Comment:
I think we already have scaleApp() function that does the same functionality. Can we replace this with scaleApp() function to reduce duplication?

Reply:
good point. Fixed.